docs/designs/2730-visibility.md

```diff
@@ -0,0 +1,293 @@
+# Improved Bootstrap Visibility
```

Comment:
What's unclear to be is whether this is for **management-cluster** bootstraping, **workload cluster** bootstrapping or both.

I think it should be for both. Would love details on what exactly we're going to scrape up or persist to provide the user details.

Reply:
oops just realized at the end:

> The main target for these changes are management-cluster and cluster create commands

I think we should call this out earlier. And I'd love some implementation details on what/how you're planning to slurp things up. I expect the `cluster` interaction to be more complicated.